pyannote/core/timeline.py

```diff
@@ -554,6 +554,87 @@ def overlapping_iter(self, t: float) -> Iterator[Segment]:
             if segment.overlaps(t):
                 yield segment
 
+    def overlaps_iter(self) -> Iterable[Segment]:
```

Comment:
Same remark: we should rename this method.

Reply:
This wasn't actually implemented (this cannot be "iterable" since we need to compute the support after finding the overlapping segments).